app/views/animals/new.html.erb

```diff
@@ -0,0 +1,18 @@
+
+
+<h1>Add the details of your animal</h1>
+
+<div class="container">
+  <%= simple_form_for(@animal,  html: { class: "form-group w-50 p-5 mx-auto"}) do |f| %>
+    <%= f.input :name %>
+    <%= f.input :species %>
+    <%= f.input :hourly_rate %>
+    <%= f.input :category, collection: Animal::CATEGORIES %>
+    <%= f.input :gender, collection: Animal::GENDERS %>
+    <%= f.input :photos, as: :file, input_html: { multiple: true } %>
+    <%= f.submit %>
+  <% end %>
```

Comment:
missing other non-mandatory input fields: description, place of origin, age

Reply:
done